host/xtest/regression_6000.c

```diff
@@ -2008,3 +2228,4 @@ ADBG_CASE_DEFINE(regression, 6017, xtest_tee_test_6017,
 		 "Test Persistent objects info");
 ADBG_CASE_DEFINE(regression, 6018, xtest_tee_test_6018, "Large object");
 ADBG_CASE_DEFINE(regression, 6019, xtest_tee_test_6019, "Storage independence");
+ADBG_CASE_DEFINE(regression, 6020, xtest_tee_test_6020, "Forbiden accesses");
```

Comment:
Nit: `s/Forbiden accesses/Object IDs in SHM (negative)/` (or at least fix the typo :))